magprime/__init__.py

```diff
@@ -42,6 +42,11 @@ class Attendee:
     def donation_cost(self):
         return self.extra_donation if self.extra_donation else 0
 
```

Comment:
Gotcha.  Yeah, there are a few other things we could do here instead.  Probably the real solution is that in cases where the value passed is invalid, we should just keep the default value instead of setting the invalid value.  This would be a simple change to ``MagModel.apply`` which I *think* would be backwards-compatible with all of our existing use cases.